man/bootctl.xml

```diff
@@ -115,7 +115,8 @@
 
         <listitem><para>Shows all available boot loader entries implementing the <ulink
         url="https://github.com/systemd/systemd/blob/master/docs/BOOT_LOADER_SPECIFICATION.md">Boot Loader
-        Specification</ulink></para></listitem>
+        Specification</ulink>, as well as all other and automatic entries the boot loader used on the current boot
```

Comment:
"all other and automatic" is grammatically suspicious and "discovered" (at the end) also seems misplaced. Maybe "as well as any other entries discovered or automatically generated by the boot loader" ?